### DISCUSSION TITLE
Sonoff ZBMINI desn't work as router for battery powred devices

### DISCUSSION THREAD

**Discussion by andreapx** [Koenkk/zigbee2mqtt#21928] [Devices]
I have a zone where the battery Zigbee devices don't reach the router.
I've added a Snoff ZBMINI switch in between that works perfectly, but the batery devices don't connect to it and they still can't connect to the Zigbee network.
I've reset them after I've installed the Sonoff switch, but still nothing.
I've created a map and I can see that it act as a router, but only for the devices that are not battery powered. So strange!
What can I do?

**Comment by francisp2** [comment]
Did you try to pair your battery devices through this router ?

**Reply by andreapx** [reply]
If you mean to pair them near the Sonoff, yes, I've reset them and paired them about 10cm from it.
They paired successfully, but after the pairing I haven't seen them online anymore.

**Comment by deviantintegral** [comment]
I've found that even when I force paired them, devices eventually went to other routers. There was a time where I did have end devices go offline, but that could have been due to other issues too.

I ended up using https://www.zigbee2mqtt.io/devices/43076.html#enbrighten-43076 in most of my switches. It was much, much easier to physically install, and they work fine as routers.

**Reply by andreapx** [reply]
I've forced it to pair with the Sonoff ZBMINI and with a bTicino switch, but nothing, it doesn't pair.
@Koenkk any hint?
